chingu-coders/Voyage2-Turtles-02
# reorganize gulp task flow

## Purpose
* Improve the speed of `browser-sync` reloads during development.
* Its unnecessary to run build tasks at every reload of browser

## Approach/Fixes
* Remove build tasks (`html`, `js`, `css`, `assets`) from the `default` gulp task.
* Create new `build` task that can be run separately before deployment

## Testing
* Checkout this PR
* run `gulp`.
* Make some changes to files in `src` folder. See how much faster the browser reloads.
* run `gulp build` to update `dist` directory only when you need to: i.e., before making a pull request.
